crons/index.js

```diff
@@ -0,0 +1,15 @@
+const { createPool } = require('slonik');
```

Comment:
What is the purpose of this extra file? Could this be handled entirely in `index.js`?